src/main/java/edu/harvard/iq/dataverse/DataFile.java

```diff
@@ -1123,4 +1123,23 @@ private boolean tagExists(String tagLabel) {
         }
```

Comment:
I'll just leave this comment here on the first file that is changed...

Can we please get a release note?